lab-patrick/lib/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const readFiles = require('../lib/fs-module');
+const paths =['../data/one.txt', '../data/two.txt', '../data/three.txt'];
+
+readFiles(paths);
```

Comment:
Having your function invocation as ``readFiles(paths)`` isn't actually accurate. 

If you refer back to your ``lib/fs-module.js``, your exported function doesn't actually take in an arguments. Instead, you defined a `text array` at the top of the module already, and your function is reading the text from that. 

Take out the word ``paths`` passed in Line 6 and comment out  your ``paths`` array. You should get the same output.

The proper way to do this would be to reformat your ``fs-module``function to take in an ``array`` parameter. I'll reference that for you in another comment.